Packs/CommonScripts/Scripts/IPNetwork/IPNetwork.py

```diff
@@ -44,21 +44,20 @@ def cidr_command(args: Dict[str, str]) -> Tuple[str, dict, str]:
 
     """
     cidr_list = argToList(args.get('cidr'))
-    ec = {'Network': [ip_cidr(cidr) for cidr in cidr_list]}
+    ec = [ip_cidr(cidr) for cidr in cidr_list]
     markdown = tableToMarkdown(name=f'CIDR - {args.get("cidr")}',
-                               t=ec.get('Network'),
+                               t=ec,
                                removeNull=True)
-
-    return (
-        markdown,
-        ec,
-        ""
+    return CommandResults(
```

Comment:
please also change the type hints in the function return type to CommandResults as well.